src/views/BusinessCase/index.tsx

```diff
@@ -63,15 +59,13 @@ export const BusinessCase = () => {
 
   const dispatchSave = () => {
     const { current }: { current: FormikProps<BusinessCaseModel> } = formikRef;
-    if (current && current.dirty && !isSaving) {
```

Comment:
`dirty` comes from Formik. We used this as a "hack" in order to only make the save/update request if the form was dirty. It was also a bandaid to stop any unnecessary saves made by the `Autosave` component. Having this conditional in the save function caused issues when we explicitly want to save (e.g. back/next buttons and save&exit).

By removing it, a save request will always be made whenever this function is executed. It is now completely up to the `Autosave` to make sure saving is only executed when necessary.